internetarchive/files.py

```diff
@@ -257,6 +258,11 @@ def download(self, file_path=None, verbose=None, silent=None, ignore_existing=No
                 and return_responses is not True:
             os.makedirs(parent_dir)
 
+        pbar = tqdm(total=self.size, 
+                    unit='B', 
+                    unit_scale=True, 
```

Comment:
```suggestion
        pbar = tqdm(total=self.size,
                    unit='B',
                    unit_scale=True,
```
Fixes https://travis-ci.org/github/jjjake/internetarchive/jobs/673581805#L281